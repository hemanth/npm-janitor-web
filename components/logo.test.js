import React from 'react'
import Logo from './logo'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <Logo/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
