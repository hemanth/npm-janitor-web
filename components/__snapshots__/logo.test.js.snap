// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="hQbxwO"
  src="static/logo.png"
/>
`;
